wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,273 @@
+Configure the custom combo colour set.
 
-Colours
--------
+Combo colours are an important part of the beatmap's aesthetic value.
 
-![Colours section](img/SS_Colours.jpg "Colours section")
+Clicking a combo colour or clicking on `New Combo Colour` opens up the OS's color picker (as shown in the image, uploader used a Windows OS). Choose colors that compliment the background but the colours must not camouflage into the background.
 
-These sections are used mainly to assign combo colors. Combo colors are an important part of the beatmap's aesthetic value, because there will be a lot of circles and sliders in the beatmap. Clicking a combo color opens up your OS's color picker (At picture, uploader used Windows OS). Choose colors that compliment the background but the colours must not camouflage into the background. Up to eight combo colors can be used, although most maps uses four. Clicking the "Remove Combo Colour" will remove the highest numbered Combo color (in other word, reverse order 8-7-6-5-4-3-2).
+Up to eight combo colors can be used, although most beatmaps use four.
 
-The other part of this window is assigning the playfield background color, but this is usually a moot point because beatmaps can't be ranked without a background image which usually overrides this setting unless you are planning to use a storyboard which the background may be visible.
+The `Enable Custom Colours` checkbox must be checked to enable the custom combo colour set above. Unchecking it will default to either the default combo colour settings or the player's skin combo colour settings. It is generally recommended to keep it checked so the beatmap has its own unique identity in terms of combo colours. It can be safely unchecked for osu!taiko-specific and osu!mania-specific beatmaps.
 
-Storyboarding
--------------
+Clicking the `Remove Combo Colour` will remove the highest numbered combo colour (in other word, reverse order starting from 8-7-6-5-4-3-2).
 
-![Design section](img/SS_Storyboarding.jpg "Design section")
+### Playfield Background
 
+Click on the `Background Colour` to change the blank playfield's background colour.
 
-### Countdown
+A moot point to change except for storyboard that may expose the blank playfield's background.
 
-A countdown similar to the EBA and Ouendan.
+## Design
 
-You can change the speed and offset of the countdown as well.
+![Design section][SS_design]
 
-### Misc.Toggles
+### Countdown
 
-**Wide Screen Support**: will remove [sidebars/pillarboxes](https://en.wikipedia.org/wiki/Pillarbox) to the left and right side of the playfield, if the client is using any aspect ratio greater than 4:3. This is typically-automatically enabled, when you start using storyboard elements.
+Check the `Enable countdown` checkbox to enable the countdown animation cheer before the timing of the first hit object similar to EBA and Ouendan games.
```

Comment:
before the timing of the first hit object similar to EBA and Ouendan games. -> before the beatmap starts